app/src/main/java/ca/marshallasch/veil/database/Database.java

```diff
@@ -109,16 +108,16 @@ public void onCreate(SQLiteDatabase db)
     {
         db.execSQL(BlockContract.SQL_CREATE_BLOCK_USERS);
         db.execSQL(NotificationContract.SQL_CREATE_POST_NOTIFICATION);
-        db.execSQL(KnownHashesContract.SQL_CREATE_KNOWN_HASHES);
         db.execSQL(UserContract.SQL_CREATE_USERS);
+        db.execSQL(KnownPostsContract.SQL_CREATE_KNOWN_HASHES);
 
     }
 
     @Override
     public void onUpgrade(SQLiteDatabase db, int oldVersion, int newVersion)
     {
```

Comment:
leave in the old migration code, just add in the new migrations

Reply:
Resolved